driver-scala/src/main/scala/org/mongodb/scala/AggregateObservable.scala

```diff
@@ -118,6 +118,33 @@ case class AggregateObservable[TResult](private val wrapped: AggregatePublisher[
     this
   }
 
+  /**
+   * Sets the comment to the aggregation. A null value means no comment is set.
```

Comment:
Copy pasta

No need for `<p>` tags in scaladoc.

Reply:
Done